filters/openpolicyagent/openpolicyagent.go

```diff
@@ -37,6 +40,7 @@ const (
 	defaultReuseDuration       = 30 * time.Second
 	defaultShutdownGracePeriod = 30 * time.Second
 	DefaultCleanIdlePeriod     = 10 * time.Second
+	DefaultMaxBodySize         = 128 * 1024 * 1024
```

Comment:
This feels to much. What is the rule of thumb to estimate this?

Reply:
I suggest we have no default constant, have flag value zero and configure actual value in the deployment manifest.

Reply:
We could also make it lower case, private, and change it later. I agree that right now 128MB seems to be a bit big for a body.

Reply:
Would 10 MB be fine?

Reply:
I have to admit I got carried away and did not see it when reviewing my changes. 128KB is what I intended to actually use. This is based on internal use cases that we looked at where 60-100KB of JSON are valid cases so far. 
I think the max header size is currently 1MB which we could also default to and would already be generous. 10MB is already strange for an authz use case IMO.